src/scene/scene_lua.inl

```diff
@@ -908,4 +908,28 @@ int addInterestingTime(lua_State* L) {
     return 0;
 }
 
+int worldPosition(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 1, "lua::worldPosition");
+
+    std::string identifier = ghoul::lua::value<std::string>(L, 1, ghoul::lua::PopValue::Yes);
+
+    glm::dvec3 pos = global::renderEngine->scene()->sceneGraphNode(identifier)->worldPosition();
```

Comment:
If the scene graph node doesn't exists, this will call the `worldPosition()` function on a nullptr and lead to an exception. To handle this, first get the pointer and then have a check that returns a lua error. For example: 

```
const SceneGraphNode* node = sceneGraphNode(identifier);
if (!node) {
    return ghoul::lua::luaError(L, "fmt::format("Did not find a match for identifier: {} ", identifier));
}
 ```
 
 (the `sceneGraphNode(string)` function, without having to access the scene pointer, is a helper that exists in `openspace/query/query.h`)

Reply:
Or looking at the previous functions in this file, maybe handle errors the same way as done there: 
```
    if (!node) {
        LERRORC(
            "worldPosition",
            fmt::format("Did not find a match for identifier: {} ", identifier)
        );
        return 0;
    }
```

Do we have any preference here, @alexanderbock ?